ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vlans.j2

```diff
@@ -31,6 +31,13 @@
 {%                 endif %}
       redistribute_routes:
         - learned
+{%                 if svi.bgp.raw_eos_cli is arista.avd.defined %}
+      eos_cli: |
+        {{ svi.bgp.raw_eos_cli | indent(8,false) }}
+{%                 endif %}
+{%                 if svi.bgp.structured_config is arista.avd.defined %}
```

Comment:
structured_config should implement inheritance from svi_profile and parent_profile (use default filter to avoid recursive merging)

Reply:
Also add test coverage for the same.

Reply:
This are will need other changes (in logic template) when #1941 has merged, so I think we should postpone this one until after #1941 and then handle the inheritance in logic.j2

Reply:
1941 has merged, so to be consistent, we should not use the merged copy but set it using default filter. I suggest to do it in logic.j2 to avoid having to parse profiles here.